### DISCUSSION TITLE
Openvpn Speed Issues

### DISCUSSION THREAD

**Discussion by saadabbasi9d** [angristan/openvpn-install#1355] [Help]
I am using Single client in openvpn but it's speed is not exceeding 1GB however i have 3 GB bandwidth speed through speedtest. 
What could be the possible reasons as there is no speed limitations and there is no error in log file . I tried this on ubuntu 20 and 22. Both have same issue.

**Comment by imayoda** [comment]
could be a CPU limit, ovpn is somehow heavyload.. try on another computer and see the difference
